src/app/components/tag/TagInput.js

```diff
@@ -107,7 +118,6 @@ class TagInput extends React.Component {
             placeholder={this.props.intl.formatMessage(messages.search)}
           />
           <Button
-            style={{ marginLeft: 'auto' }}
```

Comment:
You can put a `<Box ml="auto">` around `<Button>` to achieve the same effect while still complying with the guidelines of the ticket.